ci-operator/config/openshift/etcd/openshift-etcd-openshift-4.14.yaml

```diff
@@ -66,6 +79,14 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws-etcd-recovery
+- as: e2e-hypershift
```

Comment:
I'd make this required

Reply:
I'm fine, if they'd pass. But they currently are not, is this known?